.github/workflows/python-linting.yml

```diff
@@ -41,4 +41,11 @@ jobs:
       # Run our linting
       - name: Lint code
         run: |
-          ./bin/task lint
\ No newline at end of file
+          ./bin/task lint
+
+      # Testing
+      - name: Test code
+        env:
+          TNT_EX1_OPENWEATHERMAP_API_KEY: ${{ secrets.TNT_EX1_OPENWEATHERMAP_API_KEY}}
+        run:
+          ./bin/task test
```

Comment:
Please add a newline at the end of the file since some tools don't execute the last line if it ends with EOF (end of file) instead of a newline `\n`.